manuscript.md

```diff
@@ -25,7 +25,7 @@ and time [@Poisot2015SpeWhy]. Different types of interactions vary in their
 intrinsic predictability (*e.g.* some fungal species engage in opportunistic
 saprotrophy [@Smith2017GroEvi], obligate parasites are more deterministic
 in their interactions than facultative parasites [@Poisot2013FacObl;
-@Luong2019FacPar]). In addition to this variance in predictability, networks
+@Luong2019FacPar]. In addition to this variance in predictability, networks
```

Comment:
that should stay - it looks weird but its bracket pair is on line 25.

Reply:
sanity check me  on this one though.

Reply:
yeah